build.gradle

```diff
@@ -141,7 +144,7 @@ project.ext {
         slf4j: "org.slf4j:slf4j-api:1.7.7",
         snappy: "org.xerial.snappy:snappy-java:1.1.2.4",
         sqlite: "org.xerial:sqlite-jdbc:3.8.7",
-        sun_tools: project.files("${System.properties['java.home']}/../lib/tools.jar"),
+        //sun_tools: project.files("${System.properties['java.home']}/../lib/tools.jar"),
```

Comment:
why is this commented out?

Reply:
tools.jar doesn't exist anymore and it doesn't seem to be needed - I will delete the line